enterprise/dev/ci/internal/ci/operations.go

```diff
@@ -557,6 +557,13 @@ func triggerReleaseBranchHealthchecks(minimumUpgradeableVersion string) operatio
 func codeIntelQA(candidateTag string) operations.Operation {
 	return func(p *bk.Pipeline) {
 		p.AddStep(":docker::brain: Code Intel QA",
+			bk.SlackStepNotify(&bk.SlackStepNotifyConfigPayload{
+				Message:     ":alert: :noemi-handwriting: Code Intel QA Flake detected <@U018WMLUL3F>",
```

Comment:
```suggestion
				Message:     ":alert: :noemi-handwriting: Code Intel QA Flake detected <@code-intel-support>",
```

The plugin handles that for us.

Reply:
This pings me directly, not code-intel-support. Id rather I get paged directly :slightly_smiling_face:

Reply:
Ack, let's put your nick there then 😊

Reply:
Does that work? Not sure how/if it works given theres a space in it. ID seems to the most likely to be correct :thinking:

Reply:
https://sourcegraph.slack.com/archives/C02L31KTHHT/p1657714533415209 Yeah it does! 

```
			bk.SlackStepNotify(&bk.SlackStepNotifyConfigPayload{
				Message:              "CodeIntelQA :brain:,  checking if spaces work cc <@Noah S-C>",
```

You're right though, it should be possible to pass slack uids instead. I'll patch the plugins code along with the rest after this PR. 

For the time being, it doesn't work with uids, so for now 

```suggestion
				Message:     ":alert: :noemi-handwriting: Code Intel QA Flake detected <@Noah S-C>",
```
is the only option.